app/styles/ui/_diff.scss

```diff
@@ -36,6 +36,7 @@
       // Add a little bit of space to the left of code diff
       // `padding-left` here means mouse move events are not raised
       padding-left: var(--spacing-half);
+      align-self: center;      
```

Comment:
This fixes the vertical alignment of text on the right hand side of the gutter

**Before**
![image](https://cloud.githubusercontent.com/assets/634063/26357629/2be545b0-3fd0-11e7-857e-4aaa57b53cac.png)

**After**

![image](https://cloud.githubusercontent.com/assets/634063/26357635/31cb6cb6-3fd0-11e7-861b-7808bcc2d9af.png)